modules/ROOT/pages/vantagecloud-lake-demo-jupyter-docker.adoc

```diff
@@ -0,0 +1,126 @@
+= Run VantageCloud Lake Demos from a Jupyter notebook in Docker
+:experimental:
+:page-author: Vidhan Bhonsle
+:page-email: vidhan.bhonsle@teradata.com
+:page-revdate: January 10th, 2024
+:description: Run Teradata VantageCloud Lake Demos from Jupyter in a Docker
+:keywords: data warehouses, compute storage separation, teradata, vantage, cloud data platform, business intelligence, enterprise analytics, jupyter, teradatasql, ipython-sql, docker, container, vantagecloud, vantagecloud lake, lake
+
+== Overview
+In this how-to we will go through the steps for connecting to TeradataCloud Lake and run demos from a Jupyter notebook in Docker. 
+
+== Prerequisites
+* https://www.docker.com/products/docker-desktop/[Docker Desktop] installed
+* (Optional) https://git-scm.com/book/en/v2/Getting-Started-Installing-Git[Git] installed
+** Required to download git repo from https://github.com/Teradata/lake-demos.git. You can also download the repo as a .zip if you don't have Git installed locally.
+* A Teradata VantageCloud Lake account login
+** Organization URL and login details from Teradata welcome letter
+* (Optional) IDE of your choice
+
+== Create VantageCloud Lake environment
+Follow the instructions from the https://quickstarts.teradata.com/getting-started-with-vantagecloud-lake.html[VantageCloud Lake getting started] to create your own environment. +
+Once created, go to *SETTINGS* tab and provide your IP address range in CIDR format (for example, 192.168.2.0/24 specifies all IP addresses in the range: 192.168.2.0 to 192.168.2.255) to access the environment.
```

Comment:
Can we assume that people will know what CIDR is? Also, since this is about docker, they will likely run it on their local. Will they know how to identify their external IP address as seen by the Lake infrastructure? Should we not help them with these tasks at least by linking to some websites that explain CIDR and help identify the public IP address?